airflow/www/views.py

```diff
@@ -213,6 +218,18 @@ def get_int_arg(value, default=0):
 
         arg_current_page = request.args.get('page', '0')
         arg_search_query = request.args.get('search', None)
+        arg_tags_filter = request.args.getlist('tags', None)
+        flask_session.permanent = True
```

Comment:
What does this do? Convert the it from a session cookie to a permanent one?

Reply:
Yes, changed the entire flask session cookie to be permanent (with the expiration set in `PERMANENT_SESSION_LIFETIME`).

Reply:
Moved to app.py (with `@app.before_request`) since it's a global change and it makes more sense to be in the create_app func.